stdlib/runtime/string.gr

```diff
@@ -381,19 +386,23 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
             variantName
           } else {
             let comspace = ", "
-            Memory.incRef(WasmI32.fromGrain(SLEmpty))
-            let mut strings = slConsDisableGc(")", SLEmpty)
+            let lparen = ")"
+            let mut strings = slConsDisableGc(lparen, SLEmpty)
             for (let mut i = variantArity * 4n - 4n; i >= 0n; i -= 4n) {
               let tmp = toStringHelp(WasmI32.load(ptr + i, 20n), extraIndents, false)
               strings = slConsDisableGc(tmp, strings)
+              Memory.decRef(WasmI32.fromGrain(tmp))
               if (i > 0n) {
-                Memory.incRef(WasmI32.fromGrain(comspace))
                 strings = slConsDisableGc(comspace, strings)
               }
             }
             Memory.incRef(WasmI32.fromGrain(variantName))
-            strings = slConsDisableGc(variantName, slConsDisableGc("(", strings))
+            let rparen = "("
```

Comment:
is this an rparen? Look like an lparen to me?

Reply:
🤦 fixed